mm2src/portfolio/prices.rs

```diff
@@ -18,17 +17,17 @@
 //  marketmaker
 //
 
-use common::{dstr, lp, rpc_response, rpc_err_response, HyRes, SATOSHIDEN, SMALLVAL};
-use common::wio::slurp_req;
-use common::mm_ctx::{MmArc, MmWeak};
+use super::{default_pricing_provider, register_interest_in_coin_prices, InterestingCoins, PortfolioContext};
```

Comment:
Could you please roll back the changes unrelated to the actual work?

Reply:
Thanks, I missed them when I checked the changes
Done